RefactoringPracticeI/GameInventory/Program.cs

```diff
@@ -2,41 +2,28 @@
 
 namespace RefactoringExerciseI
 {
-
     public static class Program
     {
         public static void Main()
         {
             Console.WriteLine("Here we are!");
 
             IList<Item> Items = new List<Item>{
-                new Item {Name = "+5 Dexterity Vest", SellIn = 10, Quality = 20},
-                new Item {Name = "Aged Brie", SellIn = 2, Quality = 0},
-                new Item {Name = "Elixir of the Mongoose", SellIn = 5, Quality = 7},
-                new Item {Name = "Sulfuras, Hand of Ragnaros", SellIn = 0, Quality = 80},
+                new Item {Name = ItemNames.DexterityVest, SellIn = 10, Quality = 20},
```

Comment:
excellent! (despite the fact we usually won't be touching client code, this is good).